File: Java/Basics To Java/src/B/B15.java
package B;

import java.util.Scanner;

public class B15 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("jak je venku?");

		String odpoved = sc.next();

		switch (odpoved) {
		case "zima":
			System.out.println("se obleè..");
			break;
		case "vedro":
			System.out.println("se vysleè..");
			break;
		}
	}
}

File: C#/Basic logic/B/B15.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace B
{
    internal class B15
    {
        static void Mainx(string[] args)
        {
            Console.WriteLine("jak je venku?");
            string odpoved = Console.ReadLine();

            switch (odpoved)
            {
                case "zima": Console.WriteLine("se obleč..");break;
                case "vedro": Console.WriteLine("se vysleč..");break;
            }
        }
    }
}
